docs/source/2018Michael.rst

```diff
@@ -116,11 +138,33 @@ Hurricane Track and Intensity
     * The maximum surface wind speed at the vortex center in GFS.v16.0.10 reaches to 90 kts, which is 50 kts lower than Best Track data.
     * The minimum sea level pressure from GFS.v16.0.10 is 950 hPa, versus 919 hPa from Best Track data. 
 
+  .. group-tab:: SRW.v1.0
+
+    .. figure:: images/2018Michael/tracker_Michael_srwv1.png
```

Comment:
You can crop/trim the white spaces of the figure. It can make the figures stand out clearly. The same applies to other figures.

Reply:
Updated, thanks!